.github/scripts/release-artifacts.sh

```diff
@@ -34,9 +35,9 @@ artifacts_tag="$target_ref_short-artifacts"
 git fetch origin "$target_ref"
 git checkout --detach "$target_ref"
 
-forge build -o artifacts
+bash "$script_path/generate-artifacts.sh"
 
-git add artifacts
+git add --all
 git commit -m "Add artifacts for $target_ref"
 git tag -m "Artifacts for $target_ref" --end-of-options "$artifacts_tag"
 
```

Comment:
I didn't realize we could commit to the repo from within GitHub actions! This is great news.